salt/modules/cmdmod.py

```diff
@@ -492,6 +493,9 @@ def _get_stripped(cmd):
         run_env = os.environ.copy()
         run_env.update(env)
 
+    if prepend_path:
+        run_env['PATH'] = prepend_path + ':' + run_env['PATH']
```

Comment:
This can be done a bit more efficiently using:

```python
run_env['PATH'] = ':'.join((prepend_path, run_env['PATH']))
```

Reply:
ok, that's nice. I assume though, that you mean to replace the 2nd line you highlighted? (so the `if prepend_path` test still needs to be there)

Reply:
FYI I've added https://github.com/saltstack/salt/pull/44300/commits/f59271f17268915c1b032c904ecfd3b7d208e68f. Let me know if this works for you.

Reply:
Actually thinking about it though, I do like the original version, if only for clarity :). I've learnt a neat trick though, with this `':'.join()` thing. If you're ok just leaving it as is, I would be happy to go back to `prepend_path + ':' + run_env['PATH']` :) Let me know.